packages/cloud-cognitive/src/components/NotificationsPanel/NotificationsPanel.js

```diff
@@ -38,6 +38,7 @@ export let NotificationsPanel = React.forwardRef(
       className,
       data,
       daysAgoText,
+      defaultToggled,
```

Comment:
I suggest we make this `doNotDisturbDefaultToggled`, by analogy with `doNotDisturbLabel`, so it's clear what it is that will (or won't) be set toggled.

```suggestion
      doNotDisturbDefaultToggled,
```

Reply:
Yes, good idea

Reply:
Updated. See [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/956/commits/4a0923f5c8f74caf6dda1dc582d6b59127554de0)

Reply:
Thanks -- and I realised I missed one change :-)